x-pack/platform/plugins/shared/fleet/server/routes/remote_synced_integrations/index.ts

```diff
@@ -15,81 +15,77 @@ import { GetRemoteSyncedIntegrationsStatusResponseSchema } from '../../types/mod
 
 import { GetRemoteSyncedIntegrationsInfoRequestSchema } from '../../types';
 
-import { canEnableSyncIntegrations } from '../../services/setup/fleet_synced_integrations';
-
 import {
   getRemoteSyncedIntegrationsStatusHandler,
   getRemoteSyncedIntegrationsInfoHandler,
 } from './handler';
 
 export const registerRoutes = (router: FleetAuthzRouter) => {
-  if (canEnableSyncIntegrations()) {
-    router.versioned
-      .get({
-        path: REMOTE_SYNCED_INTEGRATIONS_API_ROUTES.STATUS_PATTERN,
-        security: {
-          authz: {
-            requiredPrivileges: [
-              FLEET_API_PRIVILEGES.SETTINGS.READ,
-              FLEET_API_PRIVILEGES.INTEGRATIONS.READ,
-            ],
-          },
+  router.versioned
```

Comment:
removed the `canEnableSyncIntegrations()` check because it seems `licenceService` is not available when this code runs in kibana start